tools/cabana/streams/routes.cc

```diff
@@ -32,10 +32,25 @@ class RouteListWidget : public QListWidget {
 RoutesDialog::RoutesDialog(QWidget *parent) : QDialog(parent) {
   setWindowTitle(tr("Remote routes"));
 
+  auto all_routes_widget = new QWidget;
+  auto all_routes_layout = new QVBoxLayout;
```

Comment:
You can directly attach a layout to the widget using:
`auto all_routes_layout = new QVBoxLayout(all_routes_widget)`

This eliminates the need to call all_routes_widget->setLayout(all_routes_layout); separately, making the code cleaner